src/components/VersionSelect/index.spec.tsx

```diff
@@ -2,28 +2,32 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 import { Form } from 'react-bootstrap';
 
-import { fakeVersions } from '../../test-helpers';
+import configureStore from '../../configureStore';
+import {
+  ExternalVersionsList,
+  actions as versionActions,
+} from '../../reducers/versions';
+import { fakeVersionsList } from '../../test-helpers';
 import styles from './styles.module.scss';
 
 import VersionSelect from '.';
 
 describe(__filename, () => {
-  const render = (props = {}) => {
+  const render = ({
+    label = 'select a version',
+    versions = fakeVersionsList,
+    withLeftArrow = false,
+  } = {}) => {
+    const addonId = 123;
+    const store = configureStore();
+    store.dispatch(versionActions.loadVersionsList({ addonId, versions }));
+    const versionsLists = store.getState().versions.byAddonId[addonId];
```

Comment:
I don't think this is necessary. You can just do:

```js
const versionsLists = createVersionsLists(versions);
```

Reply:
ah, yes that's much simpler!